tests/react/vanilla-utils/atomWithObservable.test.tsx

```diff
@@ -17,11 +26,14 @@ beforeEach(() => {
     performance.clearMarks = (() => {}) as any
     performance.clearMeasures = (() => {}) as any
   }
+  // suppress ErrorBoundary error log
+  consoleErrorSpy = vi.spyOn(console, 'error').mockImplementation(() => {})
```

Comment:
Nice idea. Can we only suppress the error boundary error, using some simple regex?

We have this, so maybe we should be consistent about using `spyOn`?
https://github.com/pmndrs/jotai/blob/ccba918281069f3ce003988f591389df4fefdde5/tests/react/error.test.tsx#L12

Reply:
atomWithObservable was excluded from older ts test, so it passed, but `error.test.tsx` it not:

https://github.com/pmndrs/jotai/blob/ccba918281069f3ce003988f591389df4fefdde5/.github/workflows/test-old-typescript.yml#L69

Reply:
Hmmm, maybe we give up `spyOn` and just assign `console.error = ...`?